### ISSUE TITLE
mp4box MPD file references missing RepresentationIndex file

### ISSUE THREAD

**Issue by hapatrick** [gpac/gpac#736]
I'm trying to use mp4box to create an MPEG-DASH files from a transport stream file. The resulting mpd file has a  `<RepresentationIndex>` node references file video_dash.six. video_dash.six is not generated.

Is this expected behavior? Should video_dash.six be created by mp4box?

Details:
I'm using this command:
mp4box -dash 2000 -rap -single-file -profile main -out video_klv.mpd video.ts

Command produces a single message on the console:
[DASH] invalid call to strip_output_dir

The resulting mpd file is below. The `<RepresentationIndex>` node references file video_dash.six, which does not exist.

This is using the latest version of mp4box (0.6.2-DEV-rev1305-ga3ad30d-master) on 64-bit Win7.

```
<?xml version="1.0"?>
<!-- MPD file Generated with GPAC version 0.6.2-DEV-rev1305-ga3ad30d-master  at 2017-01-11T16:37:10.177Z-->
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" minBufferTime="PT1.500S" type="static" mediaPresentationDuration="PT0H0M29.997S" maxSegmentDuration="PT0H0M2.069S" profiles="urn:mpeg:dash:profile:mp2t-main:2011">
 <ProgramInformation moreInformationURL="http://gpac.io">
  <Title>video.mpd generated by GPAC</Title>
 </ProgramInformation>

 <Period duration="PT0H0M29.997S">
  <AdaptationSet segmentAlignment="true" maxWidth="1920" maxHeight="1080" par="1:1">
   <Representation id="1" mimeType="video/mp2t" width="1920" height="1080" codecs="avc1.420029" startWithSAP="1" bandwidth="14588582">
    <BaseURL>video_dash1.ts</BaseURL>
    <SegmentList timescale="90000" duration="180000" presentationTimeOffset="126000">
     <RepresentationIndex sourceURL="video_dash.six"/>
     <SegmentURL mediaRange="0-3847983"/>
     <SegmentURL mediaRange="3847984-7921943"/>
     <SegmentURL mediaRange="7921944-11293723"/>
     <SegmentURL mediaRange="11293724-15509999"/>
     <SegmentURL mediaRange="15510000-18723107"/>
     <SegmentURL mediaRange="18723108-22929795"/>
     <SegmentURL mediaRange="22929796-26190467"/>
     <SegmentURL mediaRange="26190468-30065523"/>
     <SegmentURL mediaRange="30065524-33254943"/>
     <SegmentURL mediaRange="33254944-37126051"/>
     <SegmentURL mediaRange="37126052-40251927"/>
     <SegmentURL mediaRange="40251928-44452787"/>
     <SegmentURL mediaRange="44452788-47713459"/>
     <SegmentURL mediaRange="47713460-51566143"/>
     <SegmentURL mediaRange="51566144-54701043"/>
    </SegmentList>
   </Representation>
  </AdaptationSet>
 </Period>
</MPD>
```

**Comment by AdithyanIlangovan** [comment]
- I have the exact same issue. When I try to create an .mpd with the following command : 

> MP4Box -dash 2000 -rap -single-file -profile main -out parkRun.mpd parkrun1920_12mbps.ts

- I see the following .mpd being generated.

```xml
<?xml version="1.0"?>
<!-- MPD file Generated with GPAC version 0.7.2-DEV-rev336-g0a8d5cc74-master  at 2017-11-30T15:19:45.070Z-->
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" minBufferTime="PT1.500S" type="static" mediaPresentationDuration="PT0H0M0.000S" profiles="urn:mpeg:dash:profile:mp2t-main:2011">
 <ProgramInformation moreInformationURL="http://gpac.io">
  <Title>parkRun.mpd generated by GPAC</Title>
 </ProgramInformation>

 <Period>
  <AdaptationSet segmentAlignment="true" maxWidth="1920" maxHeight="1080" par="1:1">
   <ContentComponent id="49" contentType="video" />
   <ContentComponent id="50" contentType="audio" />
   <Representation id="1" mimeType="video/mp2t" width="1920" height="1080" audioSamplingRate="48000" startWithSAP="1" bandwidth="0">
    <BaseURL>parkrun1920_12mbps_dash1.ts</BaseURL>
    <SegmentList timescale="90000" duration="180000">
     <RepresentationIndex sourceURL="parkrun1920_12mbps_dash.six"/>
     <SegmentURL mediaRange="0-113957515"/>
    </SegmentList>
   </Representation>
  </AdaptationSet>
 </Period>
</MPD>
```

- Along, with the .mpd I also get an parkrun1920_12mbps_dash1.ts file generated in my directory. 

- But I do not see any parkrun1920_12mbps_dash.six file being generated.

- It is mentioned #744 already fixes this. I use the latest version, but still the problem persists.

- Could you please let me know if this is already fixed?

**Comment by jeanlf** [comment]
this has been fixed in GPAC 0.9.0
